README.md

```diff
@@ -127,20 +129,17 @@ Streams has multiple sub-projects, but you can run all the tests:
 ### Building IDE project ####
 *Note Please ensure that JDK17 is used when developing Kafka.*
 
-*Note that this is not strictly necessary (IntelliJ IDEA has good built-in support for Gradle projects, for example).*
+IntelliJ supports Gradle natively and it will automatically check Java syntax and compatibility for each module, even if
+the Java version shown in the `Structure > Project Settings > Modules` may not be the correct one.
+
+When it comes to Eclipse, run:
 
     ./gradlew eclipse
-    ./gradlew idea
```

Comment:
maybe we can remove idea plugin from build.gradle?

https://github.com/apache/kafka/blob/trunk/build.gradle#L35

Reply:
This is a larger change as it would break someone who is using this plugin in their workflow (for whatever reason). I'd prefer to start by removing it from the `README` and leaving the actual plugin removal for later. Is that reasonable?